resources/ory/profile-production.yaml

```diff
@@ -44,10 +45,11 @@ oathkeeper:
   deployment:
     resources:
       limits:
-        memory: "512Mi"
+        cpu: 10m
```

Comment:
CPU limit is not set on purpose, we should remove our artificial CPU limit setting of 10 cores in runtime-overrides. Feel free to revert this change.